circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  override:
+    - docker info
+
+    # create a version.json
+    - >
+        printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n'
+        "$CIRCLE_SHA1"
+        "$CIRCLE_TAG"
+        "$CIRCLE_PROJECT_USERNAME"
+        "$CIRCLE_PROJECT_REPONAME"
+        "$CIRCLE_BUILD_URL"
+        > version.json
+
+    - docker build -t sops:build .
+
+deployment:
+  master:
+    branch: "master"
+    commands:
+      - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
Could this leak the credentials in the CircleCI console? cc @g-k

Reply:
Yeah, when login fails it can print the user and password in the error message.

The shell script and docker v2 config in https://github.com/mozilla-services/tecken shouldn't have that problem.